src/main/java/core/basesyntax/service/impl/FruitStorageServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitStorageService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitStorageServiceImpl implements FruitStorageService {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition.